addons/web/static/src/core/color_picker/color_picker.js

```diff
@@ -198,12 +201,47 @@ export class ColorPicker extends Component {
     }
 
     getDefaultColorSet() {
-        for (const color of Object.values(this.DEFAULT_GRAYSCALES_COLORS).flat()) {
-            if (getCSSVariableValue(color) === this.state.currentCustomColor) {
-                return color;
```

Comment:
This code was introduced in the first commit and removed here (why ?). That seems strange, each commit should work correctly on its own. There should be only one commit in your PR if you want.